.github/workflows/build_base.yml

```diff
@@ -10,25 +10,34 @@ on:
         required: true
         default: ''
         type: string
+      ARCHITECTURE:
+        description: 'The architecture to build the package for'
+        required: true
+        default: 'x64'
+        type: string
 
   workflow_dispatch:
     inputs:
       CHECKOUT_TO: # This is the branch to checkout to. Defaults to 'master'
         description: 'The branch/tag/commit to checkout to'
         required: true
         default: ''
+      ARCHITECTURE:
+        description: 'The architecture to build the package for'
+        required: true
+        default: 'x64'
+        type: string
```

Comment:
**nitpick**: it could be of type `choice` instead of a `string`

Reply:
This was added in https://github.com/wazuh/wazuh-dashboard/pull/179, it is that way in the `workflow_call` as it will not be selected from an interface but called from a script. 
Same is for other repositories.